Formula/f/flow-control.rb

```diff
@@ -0,0 +1,33 @@
+class FlowControl < Formula
+  desc "Flow Control: a programmer's text editor"
+  homepage "https://github.com/neurocyte/flow"
+  version "0.3.2"
+  url "https://github.com/neurocyte/flow/archive/refs/tags/v#{version}.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/neurocyte/flow/archive/refs/tags/v0.3.2.tar.gz"
```

Reply:
Doesnt the version variable make sense? It would be easier to bump it when versions are updated? Or does this have something to do with the automatic updates logic?

Reply:
Brew can extract the version info from the URL, and it can change the version there too.

Reply:
Ah sweet! I didnt know.